cmd/nri-kubernetes/main.go

```diff
@@ -107,7 +117,7 @@ func main() {
 	}
 }
 
-func runScrapers(c *config.Config, ksmScraper *ksm.Scraper, kubeletScraper *kubelet.Scraper, i *integration.Integration, clients *clusterClients) error {
+func runScrapers(c *config.Config, ksmScraper *ksm.Scraper, kubeletScraper *kubelet.Scraper, controlplaneScraper *controlplane.Scraper, i *integration.Integration, clients *clusterClients) error {
 	if c.KSM.Enabled {
 		err := ksmScraper.Run(i)
```

Comment:
Wouldn't it make more sense to have the function in the scrappers to be called Scrape instead of Run in all the scrappers, also, I agree with the interface scrapper as Paolo suggests and I see that is marked to fix it.